### PR TITLE
Fix failing spec

### PR DESCRIPTION
**Pull request by k0kubun**
Travis builds are failing. While we should fix garage properly for kaminari v1, for now, I want to keep master builds healthy.

In addition, I upgraded Ruby versions.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,8 @@
 source "https://rubygems.org"
 
 gem "rails", ">= 4.0.0"
-gem "kaminari"
+# FIXME: Fix garage to work with kaminari v1.
+gem "kaminari", "< 1"
```
Comment (taiki45) [inline Gemfile]: Looks good but I'm wondering why this dependency was wrote here (not gemspec).

Comment (k0kubun) [inline Gemfile]: This is used by not garage but dummy rails app, isn't it? So I think it's reasonable.

Comment (taiki45) [inline Gemfile]: Hmm... I found some usages of kaminari's methods in main code (`per_page` and `page`). I'm not sure but we have to require kaminari gem as runtime dependency in this case probably?

Comment (taiki45) [inline Gemfile]: Ah, I'mt getting it. The "pagination" is an option in garage gem, so we can leave this as it is. What do you think of it?

Comment (k0kubun) [inline Gemfile]: Oops, that's true.
https://github.com/cookpad/garage/blob/8ef010e538f24db6aed1b632acc03fa157a459dc/lib/garage/paginating_responder.rb#L56

But actually it's not required for now. We can make it optional if you don't paginate resources.

Comment (taiki45) [inline Gemfile]: >actually it's not required for now

Agree :shipit:

**Pull request merged**